baseball/src/components/Display.test.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { cleanup, render, fireEvent } from "react-testing-library";
+import "jest-dom/extend-expect";
+import Display from "./Display";
+
+afterEach(cleanup);
+
+describe("<Scoreboard />", () => {
+  it("renders without crashing", () => {
+    render(<Display />);
+  });
+  it("displays strikes", () => {
```

Comment:
Great job breaking these into their own tests (strikes, balls). I think most people are just putting them into one big test but to me it makes more sense to break them out into their own tests, like you did.